CI/dartboard-deploy.groovy

```diff
@@ -0,0 +1,346 @@
+#!groovy
+// Declarative Pipeline Syntax
+@Library('qa-jenkins-library') _
+
+def agentLabel = 'jenkins-qa-jenkins-agent'
+if (params.HARVESTER_KUBECONFIG) {
+    agentLabel = 'vsphere-vpn-1'
+}
+
+def scmWorkspace
+def generatedNames
+def configDirPath
+def parseToHTML(text) { text.replace('&', '&amp;').replace('<', '&lt;').replace('>', '&gt;') }
+
+def runningContainerName
+
+pipeline {
+  agent { label agentLabel }
+
+  environment {
+    // Define environment variables here.  These are available throughout the pipeline.
+    imageName = 'dartboard'
+    qaseToken = credentials('QASE_AUTOMATION_TOKEN')
+    qaseEnvFile = '.qase.env'
+    harvesterKubeconfig = 'harvester.kubeconfig'
+    templateDartFile = 'template-dart.yaml'
+    renderedDartFile = 'rendered-dart.yaml'
+    envFile = ".env"
+    DEFAULT_PROJECT_NAME = "${JOB_NAME.split('/').last()}-${BUILD_NUMBER}"
+    accessDetailsLog = 'access-details.log'
+    summaryHtmlFile = 'summary.html'
+    s3BucketName='jenkins-bullseye-storage'
+    s3BucketRegion='us-east-2'
+  }
+
+  // No parameters block here—JJB YAML defines them
+
+  stages {
+    stage('Checkout') {
+      steps {
+        script {
+          scmWorkspace = project.checkout(repository: params.REPO, branch: params.BRANCH, target: 'dartboard')
+        }
+      }
+    }
+
+    // TODO: Set up a QASE client to utilize these for logging test run results + artifacts
```

Comment:
Make sure to add the TODO to issues here once this is merged. Some debug prints in this stage that might not be necessary, but not blocking as this stage is effectively all TODO.

Reply:
https://github.com/rancher/dartboard/issues/120